packages/trader/src/Stores/Modules/Trading/trade-store.ts

```diff
@@ -1614,11 +1613,11 @@ export default class TradeStore extends BaseStore {
         }
         const { data, event_type } = getChartAnalyticsData(state as keyof typeof STATE_TYPES, option) as TPayload;
         if (data) {
-            RudderStack.track(event_type, {
+            Analytics.trackEvent(event_type, {
                 ...data,
                 device_type: isMobile() ? 'mobile' : 'desktop',
```

Comment:
device_type already set by setAttributes function

Reply:
@NikitK-deriv this is the only place where I am sending the `device_type` 

<img width="664" alt="image" src="https://github.com/binary-com/deriv-app/assets/129206554/33d3008f-9d22-4fde-b5b0-fced3060c79f">

The above image is from deriv-analytics package, we are not setting the `device_type` in the coreData, hence we have to pass it here like this